libs/payments/ui/src/lib/server/components/UpgradePurchaseDetails/index.tsx

```diff
@@ -99,6 +100,7 @@ export function UpgradePurchaseDetails(props: UpgradePurchaseDetailsProps) {
           className="w-16 h-16 rounded-lg"
           width={64}
           height={64}
+          loader={({ src, width }) => `${src}?w=${width}&q=${75}`}
```

Comment:
Does this change have any impact on the image? Do the accounts CDN images support these additional query params?